proto/keystore_api/v1/keystore.proto

```diff
@@ -294,3 +294,17 @@ message TopicMap {
   }
   map<string, TopicData> topics = 1;
 }
+
+// A mapping of topics to their HMAC keys
+message GetConversationHmacKeysResponse {
+  // HmacKeyData wraps the HMAC key and the number of 30 day periods since epoch
+  message HmacKeyData {
+    int64 thirty_day_periods_since_epoch = 1;
```

Comment:
This could probably be an `int32` or even an `int16`, since we know it's going to be a low number. We're still in the hundreds after 50 years